composer.json

```diff
@@ -181,18 +178,18 @@
         "jms/serializer-bundle": "3.9.0",
         "symfony/doctrine-bridge": "4.4.16",
         "symfony/framework-bundle": "^4.4.38 || ^5.4.5",
-        "symfony/password-hasher": "^6.0",
-        "symfony/cache": "^6.0",
-        "symfony/amqp-messenger": "^6.0",
-        "symfony/doctrine-messenger": "^6.0",
-        "symfony/error-handler": "^6.0",
+        "symfony/password-hasher": "<5.4.0",
+        "symfony/cache": "<5.4.0",
+        "symfony/amqp-messenger": "<5.4.0",
+        "symfony/doctrine-messenger": "<5.4.0",
+        "symfony/error-handler": "<5.4.0",
```

Comment:
Why are you changing versions to be lower than 5.4? 🤔

Reply:
I don't know XD I don't know what happened here, fix incoming 🚀

Reply:
Oh, I know, these are conflicts 🤦 but I believe we can just remove them 🖖